### DISCUSSION TITLE
Distribute Multiviewer on Gentoo repositories

### DISCUSSION THREAD

**Discussion by DutchEllie** [f1multiviewer/issue-tracker#446] [Feature requests]
I would love to see Multiviewer have an official Ebuild file for Gentoo's Portage. Currently, it's only distributed as a deb, rpm and plain zip. It is possible to download and extract the zip manually every time, but it's not ideal and it would be nice to see this have an official build.

**Comment by jeppeolesen** [comment]
Hey there @DutchEllie 

Gentoo isn't on the roadmap - we are looking into improving Linux distribution using Flatpak, but there are a few impediments to overcome still before that happens.

Relevant discussions: #313 and #562
